package haptix.comm.msgs;

message hxVector3
{
  required float x = 1;
  required float y = 2;
  required float z = 3;
}
